lib/src/models/seed_changes/key_account_diffed.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:equatable/equatable.dart';
+
+import 'package:nekoton_repository/nekoton_repository.dart';
+
+/// [KeyAccountDiffed] is [KeyAccount] but without any actions.
+/// This model allows tracking KeyAccount adding/removing from [SeedList],
+/// see [SeedListDiffChange].
+class KeyAccountDiffed extends Equatable {
```

Comment:
Maybe it's better to:
* rename this class `KeyAccountDiffed` -> `KeyAccountBase`
* Extend KeyAccount from this (`KeyAccountBase`) class. As I see it contains only these fields, but we can use EquatableMixin if it should compare its own fields fields.

copying code is bad :)

Reply:
-